linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/H2Config.scala

```diff
@@ -67,6 +67,7 @@ class H2Initializer extends ProtocolInitializer.Simple {
       .replace(H2TraceInitializer.role, H2TraceInitializer.serverModule)
       .prepend(LinkerdHeaders.Ctx.serverModule)
       .prepend(h2.ErrorReseter.module)
+      .insertBefore(H2AddForwardedHeader.module.role, AddForwardedHeaderConfig.module[Request, Response])
```

Comment:
Just curious if scala can infer these type parameters if you omit them?

Reply:
Good point, I can try that out.

Reply:
So, I tried to omit the types but Scala infers that the type is `com.twitter.finagle.Stackable[com.twitter.finagle.ServiceFactory[Nothing,Nothing]]`. I tried to mess with the `module` method definition to add upper type bounds but it was not successful. The `Request` and Response` type classes for HTTP/1 and HTTP/2 do not share a parent class.

Reply:
Weird!  I would have expected type unification to be able to infer the type parameters since this is being called on a `Stack[ServiceFactory[Request, Response]]`.  Anyway, this is fine, I was just curious.